source/quick-start.txt

```diff
@@ -207,6 +207,48 @@ You can use the ``write()`` method to write objects of the following types to Mo
    write(coll, arrow_table)
    write(coll, ndarrays)
 
+You can add an optional argument to ``write()`` called ``exclude_none``.
+This argument is ``False`` by default. When it is set to ``True``, empty
+values will not be written to the database.
+
+The code in the following example writes an Arrow ``Table`` to MongoDB twice: once
+with ``exclude_none`` defaulting to ``False``, and once with it set to ``True``.
```

Comment:
Love this example code. I would be extra clear about what's happening where. Something like this:

```suggestion
The following code example writes an Arrow ``Table`` to MongoDB twice. The
first call to the ``write()`` method omits the ``exclude_none`` parameter,
so the empty field ``'b'`` is written to the database. The second call to the
``write()`` method sets ``exclude_none=True``, so the empty field ``'b'``
is ignored.
```